ConsoleApp1/Snake.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Timers;
+
+namespace ConsoleApp1
+{
+	public class Snake
+	{
+		public List<Point> Body { get; private set; }
+
+		public int LastMoveKey { get; set; } = 39;
+		public DateTime LastMovePressed { get; set; } = DateTime.Now;
+		public bool FruitGotted { get; set; } = false;
+
+		public Snake()
+		{
+			Body = new List<Point>
+			{
+				new Point(1,1),
+				new Point(2,1),
+				new Point(3,1),
+			};
+		}
+
+		public void Move(object? sender, ElapsedEventArgs e)
+		{
+			if (Console.KeyAvailable)
+			{
+				var key = Console.ReadKey(true);
+				List<int> availableKeys = new()
+				{
+					65,
+					87,
+					68,
+					83,
+					37,
+					38,
+					39,
+					40
+				};
+				if (availableKeys.Contains((int)key.Key) && (DateTime.Now - LastMovePressed).Seconds >= 1)
+				{
+					LastMoveKey = (int)key.Key;
+					LastMovePressed = DateTime.Now;
+				}
+			}
+
+			Point p = new Point(Body.Last().X + 1, Body.Last().Y);
+			switch (LastMoveKey)
+			{
+				case 37:
+				case 65:
+					{
+						p = new Point(Body.Last().X - 1, Body.Last().Y);
+					}
+					break;
+				case 38:
+				case 87:
+					{
+						p = new Point(Body.Last().X, Body.Last().Y - 1);
+					}
+					break;
+				case 39:
+				case 68:
+					{
+						p = new Point(Body.Last().X + 1, Body.Last().Y);
+					}
+					break;
+				case 40:
+				case 83:
+					{
+						p = new Point(Body.Last().X, Body.Last().Y + 1);
+					}
+					break;
+			}
+
+			if (p.X == 1 || p.X == Board.Size -1 || p.Y == 0 || p.Y == 14 || CheckSelf(p))
```

Comment:
magic numbers

what if we have a size more than 15?

Reply:
yep. youre right